### PR TITLE
added share pattern for contactcompanyassociations

### PR DESCRIPTION
**Pull request by erik-leven**

### CODE REVIEW

File: templates/system.json [hunk 1]
```diff
@@ -35,11 +35,11 @@
     },
     "companycompanyassociation_delete": {
       "method": "DELETE",
-      "url": "crm/v4/objects/company/{{ entity.id }}/associations/company/{{ entity['toObjectId'] }}"
+      "url": "crm/v4/objects/contact/{{ entity['id'] }}/associations/company/{{ entity['toObjectId'] }}"
```
Comment (JonasAls89) [inline templates/system.json:38]: This should be:

"url": "crm/v4/objects/company/{{ entity['id'] }}/associations/company/{{ entity['toObjectId'] }}" and not contact for the first one.

File: templates/system.json [hunk 2]
```diff
@@ -48,11 +48,11 @@
     },
     "companycompanyassociation_lookup": {
       "method": "GET",
-      "url": "crm/v4/objects/company/{{ entity.id }}/associations/company/{{ entity['toObjectId'] }}"
+      "url": "crm/v4/objects/contact/{{ entity['id'] }}/associations/company"
```
Comment (JonasAls89) [inline templates/system.json:51]: Same here.

File: templates/system.json [hunk 3]
```diff
@@ -48,11 +48,11 @@
...
     },
     "companycompanyassociation_update": {
       "method": "PUT",
-      "url": "crm/v4/objects/company/{{ entity.id }}/associations/company/{{ entity['toObjectId'] }}"
+      "url": "crm/v4/objects/contact/{{ entity['id'] }}/associations/company/{{ entity['toObjectId'] }}"
```
Comment (JonasAls89) [inline templates/system.json:55]: Same here

**Review by JonasAls89 — COMMENTED** [review]
Some operations in the system that does not look correct any longer. We have to correct them, then we can merge this one in.

**Review by BaardBouvet — APPROVED** [review]

**Pull request merged**